core/include/mbox.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    core_mbox Mailboxes
```

Comment:
And when I read mailbox I already can see the amount of mails asking how you can configure the SMTP client so that the mail box gets filled ;)